R/min_grid.R

```diff
@@ -326,3 +326,13 @@ min_grid.pls <- fit_max_value
 #' @export min_grid.poisson_reg
 #' @rdname min_grid
 min_grid.poisson_reg <- fit_max_value
+
+
+# When `min_grid()` is applied to grids with additional columns for
+# postprocessors, we need to nest the postprocessor columns into
+# .submodels to effectively enable the submodel trick.
+# See: https://gist.github.com/simonpcouch/28d984cdcc3fc6d22ff776ed8740004e
+nest_min_grid <- function(min_grid, post_params) {
+  # TODO
+  min_grid
```

Comment:
I thiiink this will be our entry point; patch `min_grid()` output by dropping postprocessor values into the nested `.submodel` structure.

Reply:
The postprocessing object will only be available from within a workflow.  

We could make a workflow method for `min_grid()` so that we have all of the information coming from a single object. 

It might be possible to execute `min_grid()` on the model spec then generate a separate (but similar) data structure from the postprocessor, then join the two.

Reply:
I wonder if a join, like you suggest, would be the more effective way to make this happen. Right now, my theory is that we can take the output from `model_spec` `min_grid()` methods and do some dplyr/tidyr-fu based on the postprocessor parameter names.

This is gnaaaarly, though.